Reducing the size of aspen.php one step at a time...

notrodes opened the pull request:
Did the easy thing and just moved CSS. Before starting work on backend or while working on backend we should migrate javascript to separate file.

notrodes committed:
Create README.md

notrodes committed:
Cleaned up css and separated it into theme.css.

notrodes committed:
Added short discription with room to add on.

notrodes committed:
Create .gitignore

notrodes committed:
removed secret stuff

psvenk commented:
@notrodes You should remove the IntelliJ Idea data before adding them to `.gitignore`, as Git is ignoring them and therefore not touching them (so they're still in the repo). To remove the files from the PR now, you can just use GitHub's web UI to remove the files (if that's possible), or you can remove them from `.gitignore`, delete or move them, then add them back to `.gitignore` (in three separate commits).

notrodes committed:
Update README.md

Commit:
deleted:    ../.idea/.gitignore
	deleted:    ../.idea/aspen+.iml
	deleted:    ../.idea/misc.xml
	deleted:    ../.idea/modules.xml
	deleted:    ../.idea/vcs.xml
	deleted:    ../Other Files/.DS_Store
	deleted:    .DS_Store
	../.idea/
